components/function-runtimes/python38/kubeless/kubeless.py

```diff
@@ -85,13 +89,19 @@ def metrics():
     return prom.generate_latest(prom.REGISTRY)
 
 
+@app.error(500)
+def exception_handler():
+    return 'Internet server error'
```

Comment:
did you mean "internal"?

Reply:
Yea, It was old joke in compass. I will change it.